src/App.tsx

```diff
@@ -6,29 +6,29 @@ import Routes from './Router';
 import './App.css';
 
 const App: React.FC = () => {
-  return (
-    <div className="App container">
-      <Navbar fluid collapseOnSelect>
-        <Navbar.Header>
-          <Navbar.Brand>
-            <Link to="/">Scratch</Link>
-          </Navbar.Brand>
-          <Navbar.Toggle />
-        </Navbar.Header>
-        <Navbar.Collapse>
-          <Nav pullRight>
-            <LinkContainer to="/signup">
-            <NavItem>Signup</NavItem>
-            </LinkContainer>
-            <LinkContainer to="/login">
-              <NavItem>Login</NavItem>
-            </LinkContainer>
-          </Nav>
-        </Navbar.Collapse>
-      </Navbar>
-      <Routes />
-    </div>
-  );
+    return (
+        <div className="App container">
+            <Navbar fluid collapseOnSelect>
+                <Navbar.Header>
+                    <Navbar.Brand>
+                        <Link to="/">Scratch</Link>
+                    </Navbar.Brand>
+                    <Navbar.Toggle />
+                </Navbar.Header>
+                <Navbar.Collapse>
+                    <Nav pullRight>
+                        <LinkContainer to="/signup">
+                            <NavItem>Signup</NavItem>
+                        </LinkContainer>
+                        <LinkContainer to="/login">
+                            <NavItem>Login</NavItem>
+                        </LinkContainer>
+                    </Nav>
+                </Navbar.Collapse>
+            </Navbar>
+            <Routes />
+        </div>
+    );
```

Comment:
no need to change for this project, but just to let you know we are using indentSize = 2 for the project

Reply:
noted thanks